nautobot/extras/plugins/__init__.py

```diff
@@ -147,6 +149,12 @@ def ready(self):
             register_jobs(jobs)
             self.features["jobs"] = jobs
 
+        # Import metrics (if present)
+        metrics = import_object(f"{self.__module__}.{self.metrics}")
+        if metrics is not None:
+            register_metrics(metrics)
+            self.features["metrics"] = metrics
```

Comment:
Just a note that you'll want to add this to `nautobot/extras/templates/extras/plugin_detail.html`.

Reply:
Done.